implementations/kmp-json-schema-validator/app/src/main/kotlin/io/github/sourcemeta/App.kt

```diff
@@ -0,0 +1,54 @@
+package io.github.sourcemeta
+
+
+import io.github.optimumcode.json.schema.OutputCollector
+import io.github.optimumcode.json.schema.JsonSchema
+import io.github.optimumcode.json.schema.ValidationError
+import java.io.File
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonElement
+
+
+val WARMUP_ITERATIONS: ULong = 1000.toULong()
+val MAX_WARMUP_TIME: ULong = (1e9 * 10).toULong()
+
+
+fun validateAll(schema: JsonSchema, docs: List<JsonElement>): Boolean {
+  var valid = false
+  for (doc in docs) {
+    valid = valid || schema.validate(doc, OutputCollector.flag()).valid
```

Comment:
@michaelmior Are you sure this will behave correctly? After the first run, `valid` can be true, and which point Java might short circuit on the `||` and never validate?